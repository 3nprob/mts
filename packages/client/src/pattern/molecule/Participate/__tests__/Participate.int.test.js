import { defaultRender } from './Participate.int.render'

describe('<Participate />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
