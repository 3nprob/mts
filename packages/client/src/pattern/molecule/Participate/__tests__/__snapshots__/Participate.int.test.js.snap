// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participate /> Snaphot must match defaultRender 1`] = `
<div
  className="topBorder"
>
  <label
    className="label"
    htmlFor="newsletter"
  >
    <div
      className="labelSectionHeader"
    >
      <h2
        className="labelSectionHeaderH2"
      >
        Take Part
      </h2>
      <h3
        className="labelSectionHeaderH3"
      >
        Never miss an article about web development and JavaScript.
      </h3>
    </div>
    <p
      className="labelLead"
    >
      <svg
        height="20px"
        viewBox="0 0 123.811 123.811"
        width="20px"
      >
        <path
          d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
        />
      </svg>
      Join 0+ Developers
    </p>
    <p
      className="labelLead"
    >
      <svg
        height="20px"
        viewBox="0 0 123.811 123.811"
        width="20px"
      >
        <path
          d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
        />
      </svg>
      Learn Web Development with JavaScript
    </p>
    <p
      className="labelLead"
    >
      <svg
        height="20px"
        viewBox="0 0 123.811 123.811"
        width="20px"
      >
        <path
          d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
        />
      </svg>
      Tips and Tricks
    </p>
    <p
      className="labelLead"
    >
      <svg
        height="20px"
        viewBox="0 0 123.811 123.811"
        width="20px"
      >
        <path
          d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
        />
      </svg>
      Access Tutorials, eBooks and Courses
    </p>
    <p
      className="labelLead"
    >
      <svg
        height="20px"
        viewBox="0 0 123.811 123.811"
        width="20px"
      >
        <path
          d="M29.85 106.463c3.2 3.2 7.5 5 12 5s8.8-1.8 12-5l64.799-64.8c6.5-6.5 7-17.2.7-23.9-6.6-7.1-17.7-7.2-24.399-.4l-46 46c-3.9 3.9-10.2 3.9-14.1 0l-5.9-6c-6.6-6.6-17.4-6.6-24 0-6.6 6.601-6.6 17.4 0 24l24.9 25.1z"
        />
      </svg>
      Personal Development as a Software Engineer
    </p>
    <div
      style={
        Object {
          "marginBottom": "20px",
        }
      }
    />
    <form
      className="labelForm"
    >
      <input
        autoComplete="email"
        className="labelFormInput"
        id="newsletter"
        placeholder="Your email address"
        type="email"
      />
      <button
        className="labelFormButton"
        type="submit"
      >
        Subscribe
      </button>
    </form>
    <small>
      <small>
        View ourÂ 
        <a
          href="/legal"
          itemProp="url"
        >
          Privacy Policy
        </a>
        .
      </small>
    </small>
  </label>
</div>
`;
